packages/design-system/src/components/Navigations/MenuBar/MenuBar.tsx

```diff
@@ -0,0 +1,113 @@
+import { useCallback, useEffect, useState } from 'react';
+import styled from '@emotion/styled';
+import { colors } from '../../../styles/colors';
+import { Icon } from '../../Base/Icon';
+import { VStack } from '../../Layout/StackContainer';
+
+const Component = styled.div`
+  padding: 30px 5px;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  width: 64px;
+  height: 100vh;
+  background-color: ${colors.grey12};
+  position: absolute;
+  left: 0;
+  top: 0;
+  bottom: 0;
+`;
+
+interface ProfileProps {
+  src: string;
+}
+
+const Profile = styled.div<ProfileProps>`
+  width: 40px;
+  height: 40px;
+  background-image: url(${(props) => props.src});
+  background-size: cover;
+  background-position: center;
+`;
+const Nav = styled.nav`
+  margin-top: 32px;
+  display: flex;
+  flex-direction: column;
+`;
+const Button = styled.button`
+  &:hover {
+    svg {
+      color: ${colors.black};
+    }
+  }
+`;
+
+export interface MenuBarProps {
+  profileSrc: string;
+}
+
+type MenuType = 'UilEstate' | 'UilBell' | 'UilUser' | 'UilPen';
+
+const MenuBar = (props: MenuBarProps) => {
+  const { profileSrc } = props;
+  const [menu, setMenu] = useState<MenuType>('UilEstate');
+  const onClick = useCallback((iconName: MenuType) => {
+    setMenu(iconName);
+  }, []);
+  useEffect(() => {
+    // 각 메뉴에 따른 API 호출
+    console.log(menu);
+  }, [menu]);
+  return (
+    <Component>
+      <Profile src={profileSrc} />
+      <Nav>
+        <VStack gap={20}>
```

Comment:
이 Nav 컴포넌트는 불필요해 보입니다.
VStack에 margin 스타일을 추가해주면 되어서요. 아래는 예시고, 실제 스타일 선언은 모듈 파일에서 해주셔야 합니다.
```suggestion
      <VStack css={css`margin-top: 32px`} gap={20}>
```